ironfish/src/rpc/routes/workers/getStatus.ts

```diff
@@ -55,15 +68,42 @@ router.register<typeof GetWorkersStatusRequestSchema, GetWorkersStatusResponse>(
     const jobs = getJobs(node.workerPool)
 
     if (!request.data?.stream) {
-      request.end({ started: node.workerPool.started, jobs })
+      request.end({
+        started: node.workerPool.started,
+        workers: node.workerPool.workers.length,
+        executing: node.workerPool.executing,
+        queued: node.workerPool.queued,
+        capacity: node.workerPool.capacity,
+        change: MathUtils.round(node.workerPool.change?.rate5s ?? 0, 2),
+        speed: MathUtils.round(node.workerPool.speed?.rate5s ?? 0, 2),
+        jobs,
+      })
       return
     }
 
-    request.stream({ started: node.workerPool.started, jobs })
+    request.stream({
+      started: node.workerPool.started,
+      workers: node.workerPool.workers.length,
+      executing: node.workerPool.executing,
+      queued: node.workerPool.queued,
+      capacity: node.workerPool.capacity,
+      change: MathUtils.round(node.workerPool.change?.rate5s ?? 0, 2),
+      speed: MathUtils.round(node.workerPool.speed?.rate5s ?? 0, 2),
+      jobs,
+    })
 
     const interval = setInterval(() => {
       const jobs = getJobs(node.workerPool)
-      request.stream({ started: node.workerPool.started, jobs })
+      request.stream({
+        started: node.workerPool.started,
+        workers: node.workerPool.workers.length,
+        executing: node.workerPool.executing,
+        queued: node.workerPool.queued,
+        capacity: node.workerPool.capacity,
+        change: MathUtils.round(node.workerPool.change?.rate5s ?? 0, 2),
+        speed: MathUtils.round(node.workerPool.speed?.rate5s ?? 0, 2),
+        jobs,
+      })
```

Comment:
Can we move this logic into getJobs() and rename it so it isn't duplicated in 3 spots? Kind of like how we do in getStatus() here? https://github.com/iron-fish/ironfish/blob/master/ironfish/src/rpc/routes/node/getStatus.ts#L150-L194

Reply:
Fixe3d.